### PR TITLE
Adding Container Example Start/Creation Script

### PR DESCRIPTION
**Pull request by dhaanpaa-lab0**
Here is an example container startup/creation script for being able to quickly start experimenting with Oracle XE in a docker format

### CODE REVIEW

**Comment by dhaanpaa-lab0** [comment]
@gvenzl I hope I got the right branch for adding the example scripts

**Comment by gvenzl** [comment]
Hey @dhaanpaa-lab0, 

Thanks for the PR!
Yes, the branch is the correct one! :)

File: examples/dkr-create-oracle-xe-server [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+DKR_IMAGE="gvenzl/oracle-xe"
+DKR_HOST_PORT=$1
+DKR_CNT_NAME=$2
+DKR_DATA_PATH=dbcxe-$DKR_CNT_NAME-dvol
+DKR_ORA_USER_APP=DAAPPUSER
+DKR_ORA_PW=`pwgen -1 16 | tr -d '\n'`
+DKR_ORA_PW_APP=`pwgen -1 16 | tr -d '\n'`
+
+echo "Docker Image ...............:'$DKR_IMAGE'"
+echo "Docker User ................:'$DKR_ORA_USER_APP'"
```
Comment (gvenzl) [inline examples/dkr-create-oracle-xe-server:18]: Should that one not say "Database App User". Docker User could be confusing to some.

File: examples/dkr-create-oracle-xe-server [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+echo "Docker Password ............:'$DKR_ORA_PW'"
+echo "Docker Password (app) ......:'$DKR_ORA_PW_APP'"
```
Comment (gvenzl) [inline examples/dkr-create-oracle-xe-server:20]: If you agree with the above, then this one should probably be changed "Database App User Password", or perhaps just "App User Password" to keep it short.
Either way, the user and the password descriptions should be kept in sync, i.e. print either "Database App User" and "Database App User Password" or just "App User" and "App User Password".

File: examples/dkr-create-oracle-xe-server [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+DKR_HOST_PORT=$1
+DKR_CNT_NAME=$2
+DKR_DATA_PATH=dbcxe-$DKR_CNT_NAME-dvol
+DKR_ORA_USER_APP=DAAPPUSER
+DKR_ORA_PW=`pwgen -1 16 | tr -d '\n'`
+DKR_ORA_PW_APP=`pwgen -1 16 | tr -d '\n'`
+
+echo "Docker Image ...............:'$DKR_IMAGE'"
+echo "Docker User ................:'$DKR_ORA_USER_APP'"
+echo "Docker Password ............:'$DKR_ORA_PW'"
```
Comment (gvenzl) [inline examples/dkr-create-oracle-xe-server:19]: Should that one not say "Database SYS Password" or "Database Password"?

**Review by gvenzl — CHANGES_REQUESTED** [review]

**Comment by dhaanpaa-lab0** [comment]
@gvenzl  Changes that have been made to the script that should hopefully make it easier for people to use and understand

**Review by gvenzl — APPROVED** [review]
Awesome, thanks so much for your contribution, @dhaanpaa-lab0!

**Pull request merged**